examples/sps/multiapp/yttria_thermoelectric/reference_residual/constant_electric_potential/micro_yttria_thermoelectrical_aehproperties_refres.i

```diff
@@ -703,5 +716,5 @@ initial_current_density=-5.6e-10 # -5.8e-10 #roughly for 1350K #nV/nm * \sigma
   perf_graph = true
   csv = true
   exodus = true
-#  checkpoint = true
+  #  checkpoint = true
```

Comment:
Rather than leaving this commented out line in this input file, please remove the line